packages/dcos-integration-test/extra/resiliency/test_aws_cf.py

```diff
@@ -9,44 +9,56 @@
 
 ENV_FLAG = 'ENABLE_RESILIENCY_TESTING'
 
-pytestmark = pytest.mark.skipif(
+skip_if_resiliency_not_enabled = pytest.mark.skipif(
     ENV_FLAG not in os.environ or os.environ[ENV_FLAG] != 'true',
     reason='Must explicitly enable resiliency testing with {}'.format(ENV_FLAG))
 
+skip_if_not_aws_stack = pytest.mark.skipif(
+    'AWS_STACK_NAME' not in os.environ,
+    reason='Must use an AWS Cloudformation run this test!')
+
+pytestmark = [skip_if_resiliency_not_enabled, skip_if_not_aws_stack]
+
 
 @pytest.fixture(scope='session')
-def dcos_launchpad(dcos_api_session):
-    """Interface for direct integration to dcos_launchpad hardware
-    Currently only supports AWS CF with AWS VPC coming soon
-    """
-    if 'AWS_STACK_NAME' not in os.environ:
-        # TODO(mellenburg): update when advanced templates are merged
-        pytest.skip('Must use a AWS Cloudformation to run test')
-    stack_name = os.environ['AWS_STACK_NAME']
+def boto_wrapper(dcos_api_session):
     aws_region = os.environ['AWS_REGION']
     aws_access_key_id = os.environ['AWS_ACCESS_KEY_ID']
     aws_secret_access_key = os.environ['AWS_SECRET_ACCESS_KEY']
-    bw = test_util.aws.BotoWrapper(aws_region, aws_access_key_id, aws_secret_access_key)
-    return test_util.aws.DcosCfSimple(stack_name, bw)
+    return test_util.aws.BotoWrapper(aws_region, aws_access_key_id, aws_secret_access_key)
+
+
+@pytest.fixture(scope='session')
+def dcos_stack(boto_wrapper):
```

Comment:
I was thinking the term dcos_stack was exclusive to AWS cloudformation as the other providers do not use the term stack, but best to be explicit (although its pretty explicit elsewhere in this file)